StRoot/StFcsWaveformFitMaker/StFcsWaveformFitMaker.h

```diff
@@ -52,35 +52,41 @@
 #define STROOT_STFCSWAVEFORMFITMAKER_STFCSWAVEFORMFITMAKER_H_
 
 #include "StMaker.h"
-#include "StFcsPulseFit.h"
+//#include "StFcsPulseFit.h"
+#include "StFcsPulseAna.h"
+
+#include "TH2.h"
 
 class StFcsCollection;
 class StFcsHit;
 class StFcsDb;
+class StFcsDbPulse;
 class TGraphAsymmErrors;
 class TGraph;
 class TCanvas;
-class TH1F;
-class TH2F;
+//class TH2F;
+//class TFile;
 
 class StFcsWaveformFitMaker : public StMaker {
 public:
     
     StFcsWaveformFitMaker(const char* name = "StFcsWaveformFitMaker");
     ~StFcsWaveformFitMaker();
-    int InitRun(int runNumber);
-    int Make();
-    int Finish(); 
-    void Clear(Option_t* option = "");
+    virtual int Init();
+    virtual int InitRun(int runNumber);
+    virtual int Make();
+    virtual int Finish(); 
+    virtual void Clear(Option_t* option = "");
```

Comment:
What you should really use here is the `override` keyword rather than `virtual`

Reply:
I tried using override and compiling with cons and it complains that symbol override is not defined. Can you provide a working example?